monitor/monitor_load.go

```diff
@@ -10,7 +10,8 @@ func monitorLoad(sr statRetriever, interval time.Duration) {
 	for {
 		high, err := checkLoadAvg(sr)
 		if (err == nil) && high {
-			controller.Reboot()
+			cmds := service.Command{}
+			cmds.Reboot()
```

Comment:
Not sure if go allows this, but a single line would be nice... service.Command{}.Reboot()

Reply:
It does... :-)